oa/INSTALL.md

```diff
@@ -0,0 +1,131 @@
+
+More information about oa/proxy in [README.md](https://github.com/Open-Network-Insight/oni-oa/tree/1.1/oa/proxy)
+
+More information about oa/components in [README.md](https://github.com/Open-Network-Insight/oni-oa/tree/1.1/oa/components)
+
+##Operational Analytics prerequisites
+
+In order to execute this process there are a few prerequisites:
+
+ 1. Python 2.7.
+ 2.  Components configuration. To find about how to configure each of the extra components included in this project
+        visit oa/components/[README.md](https://github.com/Open-Network-Insight/oni-oa/tree/1.1/oa/components).
+        This components are required to add context or extract additional information that is going to complement your 
+        original data. Each of this components are independent each other. Based on the data type some components are 
+        required or not.
+ 3. oni-ml results. Operational Analytics works and transforms Machine Learning results. The implementation of Machine Learning
+        in this project is through [oni-ml](https://github.com/Open-Network-Insight/oni-ml). Although the Operational Analytics
+         is prepared to read csv files and there is not a direct dependency between oni-oa and oni-ml, it's highly recomended
+        to have these two pieces set up together.
+         If users want to implement their own machine learning piece to detect suspicious connections they need to refer
+         to each data type module to know more about input format and schema.
+ 4. oni-setup project installed. Oni-setup project contains scripts to install hive database and the main configuration
+        file for Open Network Insights project.
+ 
+
+##Operational Analytics installation and usage
+####Installation
+ 
+ Operational Analytics installation consist on the configuration of extra modules or components and creation of a set of files.
+ Depending on the data type that is going to be processed some components are required and other components are not.
+ If users are planning to analyze the three data types supported (Flow, DNS and Proxy) then all components should be configured.
+
+ 1. Clone project oni-oa from [github repository](https://github.com/Open-Network-Insight/oni-oa) in your /home/\<user> directory 
+    or if it was already cloned go to oni-oa folder.
+    
+ 2. Add context files. Context files should go into oni-oa/context folder and they should contain network and geo localization
+    context. 
+    
+    2.1. Add a file ipranges.csv: Ip ranges file used by OA when running data type Flow. 
+         It should contain a list of ip ranges and the label for the given range, example:
+            
+            10.0.0.1,10.255.255.255,Internal
+            
+    2.2. Add a file iplocs.csv: Ip localization file used by OA when running data type Flow. 
+         Create a csv file with ip ranges in integer format and give the coordinates for each range.
+     
```

Comment:
Fixed it adding the link for the schema of the context files to the top of the paragraph :)